analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/ObjectsShouldNotBeDisposedMoreThanOnceTest.cs

```diff
@@ -57,8 +63,24 @@ public class ObjectsShouldNotBeDisposedMoreThanOnceTest
                 .AddReferences(MetadataReferenceFacade.SystemComponentModelPrimitives)
                 .Verify();
 
+        [DataTestMethod]
+        [DataRow(ProjectType.Product)]
+        [DataRow(ProjectType.Test)]
+        public void ObjectsShouldNotBeDisposedMoreThanOnce_Roslyn_VB(ProjectType projectType) =>
+            roslynVB.AddPaths("ObjectsShouldNotBeDisposedMoreThanOnce.vb")
+                .AddReferences(TestHelper.ProjectTypeReference(projectType))
+                .Verify();
+
 #if NET
 
+        [DataTestMethod]
+        [DataRow(ProjectType.Product)]
+        [DataRow(ProjectType.Test)]
```

Comment:
We don't need to re-do this verification in the other tests. The first one already ensures that we run for test code too
```suggestion
        [TestMethod]
```